pkgs/tools/text/popfile/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv, fetchzip, makeWrapper,
+perl, perlPackages,
+... }:
+
+let perlLibs = with perlPackages; [
+    perl
+    DBI
+    DBDSQLite
+    Digest
+    DigestMD5
+    HTMLTagset
+    MIMEBase64 # == MIMEQuotedPrint
+    TimeDate # == DateParse
+    HTMLTemplate
+    # IOSocketSocks # FIXME error
+    IOSocketSSL
+    NetSSLeay
+    SOAPLite
+  ];
+in
+
+stdenv.mkDerivation rec {
+  shortname = "popfile";
+  version = "1.1.3";
+  name = "${shortname}-${version}";
+
+  src = fetchzip {
+    url = "http://getpopfile.org/downloads/popfile-1.1.3.zip";
+    sha256 = "0gcib9j7zxk8r2vb5dbdz836djnyfza36vi8215nxcdfx1xc7l63";
+    stripRoot = false;
+  };
+
+  buildInputs = [ makeWrapper ] ++ perlLibs;
+  # propagatedBuildInputs = perlLibs;
+
+  phases = [ "unpackPhase" "installPhase" "patchPhase" "postInstall" ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cd $src
+    cp -r * $out/bin
+    cd $out/bin
+    chmod +x *.pl
+  '';
```

Comment:
This was my original approach, but it seems `nix-mode` breaks on the `*` in emacs for me.